### ISSUE TITLE
Routing can break with the new `authGuardLoggedInURL` feature

### ISSUE THREAD

**Issue by learnwell** [AnthonyNahas/ngx-auth-firebaseui#309]
### Bug Report or Feature Request (mark with an `x`)
```
- [x] bug report -> please search issues before submitting
- [ ] feature request
```

### OS and Version?
macOS mojave

### Versions
```
Angular CLI: 8.2.0
Node: 11.8.0
OS: darwin x64
Angular: 8.2.0
... animations, cli, common, compiler, compiler-cli, core, forms
... language-service, platform-browser, platform-browser-dynamic
... router

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.802.0
@angular-devkit/build-angular     0.802.0
@angular-devkit/build-optimizer   0.802.0
@angular-devkit/build-webpack     0.802.0
@angular-devkit/core              8.2.0
@angular-devkit/schematics        8.2.0
@angular/cdk                      8.1.2
@angular/fire                     5.2.1
@angular/flex-layout              8.0.0-beta.26
@angular/material                 8.1.2
@ngtools/webpack                  8.2.0
@schematics/angular               8.2.0
@schematics/update                0.802.0
rxjs                              6.5.2
typescript                        3.5.3
webpack                           4.38.0
```

### Repro steps
1. My repo was setup with routes as follows:
```
const routes: Routes = [
  {
    path: 'auth',
    loadChildren: () => import('./main/auth/auth.module').then(m => m.AuthModule),
  }
  ,{
    path: 'retain',
    loadChildren: () => import('./main/retain/retain.module').then(m => m.RetainModule),
    canActivate: [ LoggedInGuard ]
  }
  ,{
    path: 'create',
    loadChildren: () => import('./main/create/create.module').then(m => m.CreateModule),
    canActivate: [ LoggedInGuard ]
  }
  ,{
    path: 'connect',
    loadChildren: () => import('./main/connect/connect.module').then(m => m.ConnectModule),
    canActivate: [ LoggedInGuard ]
  }
  ,{
    path: '**',
    redirectTo: 'auth'
  }
];
```
2. This all worked upto release 3.1.0 ngx-auth-firebaseui ... before `authGuardLoggedInURL` was introduced ( in 3.2.0? )
3. But after I upgraded to the releases beyond that (3.3.0 for example) the `authGuardLoggedInURL` seemed to get in the way and I could not navigate move to any of the routes configured above other than always being stuck on the `auth` route.

I am not sure if its a corner-case or commonplace so I wanted to bring it to your attention to decide.

### The log given by the failure
There are no logs produced.

### Desired functionality
I think `authGuardLoggedInURL` should be optional and implemented in a way where it doesn't break routing or block routing circularly just because a value wasn't configured for it.

### Mention any other details that might be useful
Thanks for supporting the open source community with your library!

**Comment by AnthonyNahas** [comment]
@learnwell first of all, thanks for contributing and opening this ticket.

May you post your configuration, per example how are you injecting the library's module ?..

**Comment by AnthonyNahas** [comment]
this has been patched in [v3.3.1](https://www.npmjs.com/package/ngx-auth-firebaseui)

thanks for opening this issue

cheer 🍻
